lib/node_modules/@stdlib/math/base/special/sec/README.md

```diff
@@ -0,0 +1,215 @@
+<!--
+
+@license Apache-2.0
+
+Copyright (c) 2018 The Stdlib Authors.
```

Comment:
The standard practice is to add the current year when creating new files or packages. However, if you're only modifying existing files, the original year of creation should remain unchanged.

Reply:
Ok

Reply:
I have updated license as per year 2024